runtime/lua/vim/lsp/util.lua

```diff
@@ -877,37 +877,26 @@ end
 --- window for `textDocument/hover`, for parsing the result of
 --- `textDocument/signatureHelp`, and potentially others.
 ---
+--- Note that if the input is of type `MarkupContent` and its kind is 'plaintext',
+--- then the corresponding value is returned without further modifications.
```

Comment:
Will the plaintext be rendered as markdown? Ideally it should be treated as `filetype=text`. https://github.com/neovim/neovim/commit/20c331915f4e317c615c7cfea469a9baedd2e4f7

Reply:
No; the plan is for plaintext to be shown verbatim, without going through any styling.